src/components/EventListCard/Modal/EventListCardModals.tsx

```
@@ -97,38 +97,38 @@ function EventListCardModals({
   const [recurrence, setRecurrence] = useState<InterfaceRecurrenceRule | null>(
     eventListCardProps.recurrenceRule
       ? {
-          ...eventListCardProps.recurrenceRule,
-          endDate: eventListCardProps.recurrenceRule.recurrenceEndDate
-            ? new Date(eventListCardProps.recurrenceRule.recurrenceEndDate)
-            : undefined,
-          never: !eventListCardProps.recurrenceRule.recurrenceEndDate,
-        }
+        ...eventListCardProps.recurrenceRule,
```

ESLint

[error] Insert `··`

(prettier/prettier)